e3sm_diags/__init__.py

```diff
@@ -1,12 +1,7 @@
 import os
 import sys
 
-# import shapely here (before any esmpy imports via cdms2) to prevent a
-# segfault related to multiprocessing.  Do not import esmpy here to prevent
-# issue with dask when using ESMF with system compilers.
-import shapely
```

Comment:
FYI @xylar I don't think this is needed anymore becaues `cdms2` and `esmpy` are no longer direct dependencies. Correct me if I'm wrong though. This is related to #715.

Reply:
Sounds reasonable.